conf/modules.config

```diff
@@ -534,16 +534,17 @@ if (!params.skip_alignment) {
 
 if (!params.skip_alignment && params.aligner == 'star_salmon') {
     process {
-        withName: '.*:ALIGN_STAR:STAR_ALIGN|.*:ALIGN_STAR:STAR_ALIGN_IGENOMES' {
+        withName: 'STAR_ALIGN' {
             ext.args   = { [
+                '--alignEndsType EndToEnd',
+                '--outSAMattributes All',
                 '--quantMode TranscriptomeSAM',
                 '--twopassMode Basic',
                 '--outSAMtype BAM Unsorted',
                 '--readFilesCommand zcat',
                 '--runRNGseed 0',
                 '--outFilterMultimapNmax 20',
                 '--alignSJDBoverhangMin 1',
-                '--outSAMattributes NH HI AS NM MD',
                 '--quantTranscriptomeBan Singleend',
                 '--outSAMstrandField intronMotif',
```

Comment:
is the order of any importance?

Reply:
Of the CLI args? Don't think so

Reply:
Then why don't we order it?
(You know I like a logical order, alphabetical or any other one as long as it makes sense)

Reply:
Oh, alright. [Happy now](https://github.com/nf-core/riboseq/pull/12/commits/d7198434940ee2a23ad92cb4f3d6aa6911eddc3e)? :-D